src/wolfgame.py

```diff
@@ -1529,13 +1938,13 @@ def chk_decision(cli, force = ""):
                     # Also kill the very last person to vote them, unless they voted themselves last in which case nobody else dies
                     target = voters[-1]
                     if target != votee:
-                        if var.ROLE_REVEAL:
+                        if var.ROLE_REVEAL in ("on", "team"):
                             r1 = var.get_reveal_role(target)
                             an1 = "n" if r1.startswith(("a", "e", "i", "o", "u")) else ""
                             tmsg = ("As the noose is being fitted, \u0002{0}\u0002's totem emits a brilliant flash of light. " +
                                     "When the villagers are able to see again, they discover that \u0002{1}\u0002, " +
                                     "a{2} \u0002{3}\u0002, has fallen over dead.").format(votee, target, an1, r1)
-                        else:
+                        elif var.ROLE_REVEAL == "off":
```

Comment:
It doesn't make a functional difference currently, but this is the only place you do this, in all the other places it's just `else:`.